.github/workflows/docker.yml

```diff
@@ -1,7 +1,7 @@
 ---
 name: "Docker"
 
-on:
+on: # noqa yaml[truthy]
```

Comment:
I suggest removing this comments and simply excluding the `.github` directory from the ansible-lint checks.

Reply:
Done